### PR TITLE
Entry config

### PR DESCRIPTION
**Pull request by sndrs**
## What does this change?

- creates a webpack config per entry
- removes the entries lookup for browser/server bundles

## Why?

- means we can use the `string-replace-loader` across all files, not just the entry pages, since we can now configure the replacement string in the loader config per entry, not just the inline query for the entry itself.

this is necessary for #82, but i don't want to fill that up with more changes

### CODE REVIEW

File: webpack/index.js [hunk 1]
```diff
@@ -76,13 +94,49 @@ const common = ({ platform }) => ({
...
+module.exports = getSites().then(sites =>
+    sites.reduce(async (configs, site) => {
+        const pages = await getPagesForSite(site);
+        return [
+            // server bundle config
+            merge(
+                {
+                    entry: {
+                        [`${site}.server`]: require.resolve(
+                            `@guardian/rendering/server`,
```
Comment (SiAdcock) [inline webpack/index.js]: Does this need to be a template string?

Comment (SiAdcock) [reply webpack/index.js]: Why not `require()` at the top of the module? Looks like it's just exporting a singleton

Comment (sndrs) [reply webpack/index.js]: this just returns the path, which webpack then uses to create the bundle. i'm not sure webpack will work with the export here?

Comment (SiAdcock) [reply webpack/index.js]: Right on, I am drunk

**Pull request merged**